model-lab/app/model.py

```
@@ -1,16 +1,96 @@
 from PIL import Image
+import numpy as np
+import onnxruntime as ort
+import onnx
+from onnx import version_converter
+import cv2
 import torch
 
-MODEL_NAME = "trpakov/vit-face-expression"
+class Model:
+    #emotion table for Emotion-ferplus
+    emotion_table = {'neutral':0, 'happiness':1, 'surprise':2, 'sadness':3, 'anger':4, 'disgust':5, 'fear':6, 'contempt':7}
```

Ruff (0.14.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)